ow_materials/include/AxisAlignedGrid.tpp

```diff
@@ -0,0 +1,80 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include <cmath>
+#include <algorithm>
+
+namespace ow_materials {
+
+static void increaseToNearestMultiple(double &out_x, double const divisor) {
+  out_x += std::remainder(out_x, divisor);
+}
+
+template <typename T>
+AxisAlignedGrid<T>::AxisAlignedGrid(
+  double const x0, double const y0, double const z0,
+  double const x1, double const y1, double const z1,
+  double const side_length, T const initial_value
+) : m_side_length(side_length)
```

Comment:
I'll mention again that I think the superfluous `const` declarations should go.  A more important concern is to check for `side_length` not being 0, otherwise it gets passed as a divisor later on.

Reply:
I addressed this in the AxisAlignedGrid.h:32 comment.